docker-compose/docker-compose.yml

```
@@ -0,0 +1,34 @@
+      - NEXT_PUBLIC_DEFAULT_API_KEY=${NEXT_PUBLIC_DEFAULT_API_KEY:-exosphere@123}
+    depends_on:
+      - exosphere-state-manager
+    ports:
+      - "3000:3000"
+    networks:
+      - exosphere-network
+
+networks:
+  exosphere-network:
+    driver: bridge
+    attachable: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)